lib/widgets/compose_box.dart

```diff
@@ -1128,8 +1110,40 @@ class ComposeBox extends StatelessWidget {
     }
   }
 
+  Widget? _errorBanner(BuildContext context) {
+    final store = PerAccountStoreWidget.of(context);
+    final selfUser = store.users[store.selfUserId]!;
+    final localizations = ZulipLocalizations.of(context);
+    switch (narrow) {
+      case ChannelNarrow(:final streamId):
+      case TopicNarrow(:final streamId):
+        final channel = store.streams[streamId]!;
```

Comment:
This will crash if we find ourselves at a narrow for a stream/channel we don't know about.

We should avoid crashing in that case. For example it can happen if the user tries to follow a link that goes to a channel they don't have access to.

The principle is the same as the one described for users in #716. If you look at the other references to `store.streams` in this file, you'll see they follow that principle.